fiftyone/core/cli.py

```diff
@@ -3193,8 +3193,17 @@ def setup(parser):
             help="whether to monitor the state of the operation with a parent process",
         )
 
+        parser.add_argument(
+            "--monitor-interval",
+            type=int,
+            default=fo.config.delegated_operation_monitor_interval,
+            help="the interval in seconds at which to monitor the operation status (default: %(default)s)",
+        )
+
     @staticmethod
     def execute(parser, args):
+        if args.monitor_interval < 1:
```

Comment:
should we also check that if monitor_interval is passed that monitor is also passed?

Reply:
yeah I was thinking having both was kinda redundant and maybe remove the bool...